### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ocefpaf**
@jcermauwedu I'd prefer to send these changes to your PR 51 but it seems I cannot edit you commits. Maybe you need to check the box in:

Screenshot from 2024-05-23 10-46-19

I removed as much lint as I could so we can review your new additions. more clearly. I also preserved all your original commits here. Happy to close this an fix #51 if you prefer.

### CODE REVIEW

File: cc_plugin_glider/glider_dac.py [hunk 1]
```diff
@@ -31,7 +31,9 @@ class GliderCheck(BaseNCCheck):
     _cc_display_headers = {3: "Required", 2: "Recommended", 1: "Suggested"}
     acceptable_platform_types = {"Seaglider", "Spray Glider", "Slocum Glider"}
 
-    def __init__(self):
+    def __init__(self, options=None):
```
Comment (ocefpaf) [inline cc_plugin_glider/glider_dac.py:34]: @jr3cermak do you want a generic `options` kw or should we name this `skip_variables`? I may not be understanding what you want to do but it seems that it is the latter, right?

**Comment by jcermauwedu** [comment]
I want the variables to be checked for other requirements - so we cannot necessarily just skip whole variables.  At this point, we have datasets that do not contain QARTOD flagging.  The tests flag that `ancillary_variables` attribute as missing.   That was the intent to allow the dictionary of options through to catch various flags.   Skipping the QC checks only did certain things if I recall.  This will be something to follow up on today.

**Comment by ocefpaf** [comment]
> That was the intent to allow the dictionary of options through to catch various flags. Skipping the QC checks only did certain things if I recall. This will be something to follow up on today.

Got it. Thanks! If we can have an idea of these various flags and write up some docs I believe we can finish this one today.

**Comment by ocefpaf** [comment]
Closing this b/c I cannot update it. Somehow using gh repo checkout broke my workflow.

**Pull request closed without merging**